rllib/models/catalog.py

```diff
@@ -50,268 +50,69 @@
 # fmt: off
 # __sphinx_doc_begin__
 MODEL_DEFAULTS: ModelConfigDict = {
-    # Experimental flag.
-    # If True, user specified no preprocessor to be created
-    # (via config._disable_preprocessor_api=True). If True, observations
-    # will arrive in model as they are returned by the env.
```

Comment:
I @sven1977, I know that this pull request has been merged and that this is the old API, but why have these comments (from this key and below) been removed? Thank you in advance.